apps/etterlatte-vedtaksvurdering-kafka/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/samordning/SamordningMottattRiver.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.etterlatte.vedtaksvurdering.samordning
+
+import no.nav.etterlatte.VedtakService
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.MessageContext
+import no.nav.helse.rapids_rivers.RapidsConnection
+import org.slf4j.LoggerFactory
+import rapidsandrivers.migrering.ListenerMedLogging
+
+internal class SamordningMottattRiver(
+    rapidsConnection: RapidsConnection,
+    private val vedtaksvurderingService: VedtakService,
+) : ListenerMedLogging() {
+    init {
+        initialiserRiver(rapidsConnection, "VEDTAK:SAMORDNING_MOTTATT") {
+            validate { it.requireKey("vedtakId") }
+        }
+    }
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    override fun haandterPakke(
+        packet: JsonMessage,
+        context: MessageContext,
+    ) {
+        val vedtakId = packet["vedtakId"].asText()
+        logger.info("Behandle mottatt samordning for vedtak [vedtakId=$vedtakId]")
+
+        try {
+            val samordnetVedtak = vedtaksvurderingService.samordnetVedtak(vedtakId)
+            logger.info("Behandlet samordning ferdig for vedtak [behandlingId=${samordnetVedtak.behandling.id}]")
+        } catch (e: Exception) {
+            logger.error("Feil ved oppdatering av vedtak til SAMORDNET [vedtakId=$vedtakId]")
```

Comment:
Denne er kanskje litt skummel hvis den ikke kastes videre? Meldingen vil bli slukt, ønsker vi det hvis feks kallet med vedtaksvurdering feiler?

Reply:
> _Kunne feks brukt withLogContext pr melding_ 

Jeg forstår det slik at dette dekkes av bruken av det i _no.nav.etterlatte.kafka.startLytting_?

Reply:
Det ser ut til å være gjeldende pattern at det bare slukes i disse tilfellene, men jeg er helt med på å ikke gjøre det her inntil videre.

Når det er sagt, så har det materialisert seg et behov for å ha noe saksbehandlerstøtte for å manuelt overstyre/trigge samordningen da minst et par TP-ordninger ikke leverer svar elektronisk. Kan også være kjekt for å kunne sjekke ståa på samordningen underveis (f.eks. hvilke ordninger er meddelt, og har svart).